loaders/profane/profane.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const config = require('../../src/config/config');
```

Comment:
@obe711 is it standard practice to use relative directory paths like this? (rather than $<project path>/src/foo/foo.js)